Slugifier/SlugifierInterface.php

```diff
@@ -11,19 +11,12 @@
 
 namespace Symfony\Cmf\Bundle\CoreBundle\Slugifier;
 
+use Symfony\Cmf\Api\Slugifier\SlugifierInterface as ApiSlugifierInterface;
+
 /**
- * Slugifier interface.
- *
- * @author Daniel Leech <daniel@dantleech.com>
+ * @deprecated Since version 1.3, to be removed in 2.0. Install and use the
+ *             symfony-cmf/slugifier-api package instead.
  */
-interface SlugifierInterface
+interface SlugifierInterface extends ApiSlugifierInterface
```

Comment:
If we keep the proxy implementations in this bundle, then we are at least not breaking exisiting packages. F.e. Sulu uses the `CallbackSlugifier` and `SlugifierInterface`, if we keep these stubb implementations here then there will be no BC break, and we can move over to the new APIs when we can.

Perhaps we should just override the constructors and issue a deprecation warning?